ui/apps/platform/src/css.imports.ts

```diff
@@ -1,5 +1,5 @@
 import 'react-toastify/dist/ReactToastify.css';
-import 'app.tw.css'; // this file is the main Tailwind entrypoint handled by react-scripts
+import './app.tw.css'; // this file is the main Tailwind entrypoint handled by react-scripts
```

Comment:
Is **handled by react-scripts** redundant and therefore to delete instead of replace with **vite**?